manifests/loadbalancer.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: loadbalancer.harvesterhci.io/v1alpha1
+kind: LoadBalancer
+metadata:
+  name: cw-lb-2
+  namespace: default
+spec:
+  backendServers:
+  - 172.16.0.119
+  description: Test by cw
+  healthCheck:
+    interval: 1
+    port: 80
+    successThreshold: 1
+    failThreshold: 3
+    timeout: 1
+  listeners:
```

Comment:
Generally, the server address is included in listener attributes. Why do we have `listeners` and `backendServers` separately?

Reply:
Because for different listeners, the backend servers are the same in our case.